src/types/profile-derived.js

```diff
@@ -103,3 +105,5 @@ export type MarkerTiming = {
   length: number,
 };
 export type MarkerTimingRows = Array<MarkerTiming>;
+
+export type Implementation = 'js' | 'cpp' | 'unknown';
```

Comment:
So at this point we have a bit of ambiguity around `Implementation`. I was chatting with Markus about this, and I think we should do the following instead:

```js
export type StackType = 'js' | 'native' | 'unsymbolicated';
```

Then in the tooltip, instead of "Implementation:" it would say "Stack Type:". The human readable labels would be:

"JavaScript"
"Native"
"Unsymbolicated Native"

I think later we will also include a PseudoStack type of entry.